utils/polus-imagej-util/{{cookiecutter.project_slug}}/Dockerfile

```diff
@@ -0,0 +1,19 @@
+
+FROM labshare/polus-bfio-util:2.1.9-imagej
+
+# environment variables defined in labshare/polus-bfio-util
+# ENV EXEC_DIR="/opt/executables"
+# ENV DATA_DIR="/data"
+# ENV POLUS_EXT=".ome.tif"
+# ENV POLUS_LOG="INFO" # Change to WARNING for fewer logs, and DEBUG for debugging
+
+# Work directory defined in the base container
+# WORKDIR ${EXEC_DIR}
+
+COPY VERSION ${EXEC_DIR}
+COPY src ${EXEC_DIR}/
+
+RUN pip3 install -r ${EXEC_DIR}/requirements.txt --no-cache-dir
+
+# Default command. Additional arguments are provided through the command line
+ENTRYPOINT ["python3", "main.py"]
```

Comment:
I modified this to use the latest bfio base container. I tested a few plugins to verify that I didn't break anything.

Reply:
I updated the Dockerfile with the changes showed here.